src/components/sets/sets-manager.scss

```diff
@@ -162,4 +177,20 @@ $set-name-height: 16px;
 
 .named-set-node-menu-container {
     transition: opacity 0.25s;
+}
+
+.popover-color.twitter-picker {
+    box-shadow: none !important;
+    > div:nth-child(3) {
```

Comment:
Can you add a couple comments in the CSS? With classes I can search the names and connect the dots, but I would have some trouble tracking down what this does.